src/alignof.cc

```diff
@@ -0,0 +1,68 @@
+
+#include "env.h"
+#include "env-inl.h"
+#include "node.h"
+#include "node_internals.h"
+#include "v8-profiler.h"
+#include "v8.h"
+
+namespace node {
+namespace _alignof {
+
+using v8::Context;
+using v8::Handle;
+using v8::Object;
+using v8::Persistent;
+using v8::Uint32;
+using v8::Value;
+
+
+void Initialize(Handle<Object> exports,
+                Handle<Value> unused,
+                Handle<Context> context) {
+  Environment* env = Environment::GetCurrent(context);
+
+#define SET_ALIGNOF(name, type) \
+  struct s_##name { type a; }; \
+  exports->Set(FIXED_ONE_BYTE_STRING(env->isolate(), #name ), \
+               Uint32::NewFromUnsigned(env->isolate(), static_cast<uint32_t>(__alignof__(struct s_##name))));
```

Comment:
I forget we're in C++ land sometimes :) Does that mean I can ditch the struct thing?